module/VuFind/src/VuFind/Form/Handler/Database.php

```diff
@@ -89,12 +100,15 @@ public function handle(
     ): bool {
         $fields = $form->mapRequestParamsToFieldValues($params->fromPost());
         $fields = array_column($fields, 'value', 'name');
-
+        $userVal = null;
+        if ($user) {
```

Comment:
Maybe worth adding a comment here, like:

`// Backward compatibility: convert Laminas\Db to Doctrine; we can simplify after completing migration.`